unit-tests-java8/src/test/java/org/eclipse/collections/test/map/immutable/ImmutableMapTestCase.java

```diff
@@ -23,8 +24,9 @@ public interface ImmutableMapTestCase extends UnsortedMapIterableTestCase, Immut
     <K, V> ImmutableMap<K, V> newWithKeysValues(Object... elements);
 
     @Override
+    @Test
     default void Iterable_remove()
     {
-        UnsortedMapIterableTestCase.super.Iterable_remove();
+        ImmutableUnorderedIterableTestCase.super.Iterable_remove();
```

Comment:
Was this by design? Isnt `UnsortedMapIterable` higher than `ImmutableUnorderedIterable`